### ISSUE TITLE
Test case

### ISSUE THREAD

**Issue by dreambeam** [go-ocf/sdk#47]
I wanted to test the SDK however a readme on usage is not provided. How can I test this SDK locally?

**Comment by cizmazia** [comment]
Yes, see the [local client](https://github.com/go-ocf/sdk/tree/master/local) and related test cases.

**Comment by dreambeam** [comment]
Cool, thank you
